### DISCUSSION TITLE
resistances feedback

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#8703] [Unstable feedback]
here i rant about how damage resistances currently work (or rather don't work)

First off the amount of protection you have by default in this game is ridiculous, rendering some monsters powerless against you in a lot of situations.
https://user-images.githubusercontent.com/55637865/161391802-df14be79-a490-40ed-baed-4e8f72ee8eaf.mp4

Then there's the problem with things like body armor, which only protect the torso, which is useless since 90% of the times when you're aiming a weapon or getting shot your limbs are gonna be in front of the torso at all times.
https://user-images.githubusercontent.com/55637865/161392015-0608bfdc-a878-405e-bbc6-227f8ebe9f35.mp4

It's pretty ridiculous how tanky you can get even on early game, with the huge amounts of different resistances that stack upon eachother between drugs, talents, armor, etc.
https://user-images.githubusercontent.com/55637865/161392483-b2aee9da-c960-4070-9694-02b294e9ad20.mp4

The only thing that actually gets to kill you most of the time is bleeding, almost every armor and resistance reduces the damage taken from the variations of internal damage, but not bleeding.
(attacked by 10 tiger threshers 3 times)
![image](https://user-images.githubusercontent.com/55637865/161392558-0ba391d6-c650-41b7-a9e7-7207f7f44698.png)

So monsters that have no bleeding damage can just become harmless, even the big ones, mostly with opium items and how their effects can stack  the more you use.
https://user-images.githubusercontent.com/55637865/161392876-3a2e3bb5-f5b7-40bf-96cd-4874a7af5812.mp4

It feels like you sometimes forget 90% of the time humans are wearing some sort of protection, for example the swarmfeeders that have one bleeding damage when usually players enter alien ruins with like 40% bleeding protection.

I have a few suggestions:
Make resistances from anything other than armor way less noticeable, no 25% damage protection from vigor bs
Remove the human's natural resistances (except the bleeding resistances)
Don't allow stacking the effects of drugs

i didn't get many opinions from other people this time, but i believe most will probably agree with all of this.

**Comment by ubertpendragon** [comment]
Agreed, with the stacking of innate + talent + suit + clothes + drugs you have 5 layers of multiplicative resistances, making balance difficult.

There's also the "pirate meta" where the whole crew ends up dressed as pirates because they offer the best level of protection.
You get punished for wearing clothes for RP. Clothes should not have resistances, use the body armor/suit slot for that (engineer/mechanic gloves?), use the hat slot for small bonuses (like pirate hat)

If I'm not wrong, diving suits also protect head, arms and legs making it superior in every way to body armors, even indoors.
Body armor should apply resistances to Torso, legs and arms.
Head is kept separate for helmets.

**Comment by Ek-Videogames** [comment]
`* Laughs in abyss suit *`

Monsters suffer the same balance problems and - due to sometimes inconsistent logic regarding what kind of damage certain weapons do - end up with situations like last update where fractal guardians would be basically immune to explosions, pulse laser, and incendiary shotgun shells due to their high burn resistance.

I more or less made the same points in the discussion regarding monster "resistances & weaknesses" - having strengths and weaknesses is fine but in its current form it makes the different damage types so wildly inconsistent from enemy to enemy that it just makes combat frustrating to newer players if not using the "right meta" weaponry.

**Comment by MateusJK15** [comment]
> Then there's the problem with things like body armor, which only protect the torso, which is useless since 90% of the times when you're aiming a weapon or getting shot your limbs are gonna be in front of the torso at all times.

That could be solved by making the actual body armor into a full body armor like this :
![4395390](https://user-images.githubusercontent.com/86162808/162257284-bef2b50f-c761-42ad-8aff-b884b6f71d45.jpg)
